app/code/Magento/Sitemap/Model/Observer.php

```diff
@@ -67,13 +69,21 @@ class Observer
     protected $inlineTranslation;
 
     /**
+     * @var \Magento\Store\Model\App\Emulation $appEmulation
+     */
+    protected $appEmulation;
```

Comment:
please change the access level to private

Reply:
ok